index.html

```diff
@@ -12360,7 +12360,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="7"</strong>&gt; cantaloupes &lt;/li&gt;
   &lt;li role="option" aria-setsize="16" <strong>aria-posinset="8"</strong>&gt; dates &lt;/li&gt;
 &lt;/ul&gt;</pre>
-				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. Authors SHOULD use <pref>aria-setsize</pref>.</p>
+				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. Authors SHOULD use <pref>aria-setsize</pref>. If <pref>aria-setsize</pref> is specified, authors MUST specify a value for <pref>aria-posinset</pref>.</p>
```

Comment:
I think we should simplify this a bit and delete the "SHOULD" sentence entirely?
If we do that, would need to re-order the "If one then the other" sentence.

```suggestion
				<p>Authors MUST set the value for <pref>aria-posinset</pref> to an integer greater than or equal to 1, and less than or equal to the size of the set when that size is known. If <pref>aria-posinset</pref> is specified, authors MUST specify a value for <pref>aria-setsize</pref>.</p>
```